tests/backup/backup_basic_test.go

```diff
@@ -71,10 +113,37 @@ var _ = BeforeSuite(func() {
 	defer EndTorpedoTest()
 	// Create the first bucket from the list to be used as generic bucket
 	providers := getProviders()
-	bucketNames := getBucketName()
+	bucketNameSuffix := getBucketNameSuffix()
 	for _, provider := range providers {
-		bucketName := fmt.Sprintf("%s-%s", provider, bucketNames[0])
-		CreateBucket(provider, bucketName)
+		switch provider {
+		case drivers.ProviderAws:
+			globalAWSBucketName = fmt.Sprintf("%s-%s", globalAWSBucketPrefix, bucketNameSuffix)
+			CreateBucket(provider, globalAWSBucketName)
+			log.Infof("Bucket created with name - %s", globalAWSBucketName)
+		case drivers.ProviderAzure:
+			globalAzureBucketName = fmt.Sprintf("%s-%s", globalAzureBucketPrefix, bucketNameSuffix)
+			CreateBucket(provider, globalAzureBucketName)
+			log.Infof("Bucket created with name - %s", globalAzureBucketName)
+		case drivers.ProviderGke:
+			globalGCPBucketName = fmt.Sprintf("%s-%s", globalGCPBucketPrefix, bucketNameSuffix)
+			CreateBucket(provider, globalGCPBucketName)
+			log.Infof("Bucket created with name - %s", globalGCPBucketName)
+		}
+	}
+	lockedBucketNameSuffix, present := os.LookupEnv("LOCKED_BUCKET_NAME")
+	if present {
+		for _, provider := range providers {
+			switch provider {
+			case drivers.ProviderAws:
+				globalAWSLockedBucketName = fmt.Sprintf("%s-%s", globalAWSLockedBucketPrefix, lockedBucketNameSuffix)
+			case drivers.ProviderAzure:
```

Comment:
Object lock is more of a S3 only thing you can remove other providers.

Reply:
Done